.github/workflows/push_and_pull_crowdin_translations.yml

```diff
@@ -1,139 +1,139 @@
+
+                  # We compare the generated messages.json with the messages.json from Crowdin in p2p.
+                  # Only send a Slack message and upload it to Crowdin if there were any changes made to messages.json.
+                  if [ "$crowdin_p2p_messages_json_hash" != "$new_p2p_messages_json_hash" ]; then
+                    echo "Hashes are different, uploading to Crowdin"
+                    echo "- [crowdin]: $crowdin_p2p_messages_json_hash";
+                    echo "- [generated]: $new_p2p_messages_json_hash";
+
+                    # Upload to Crowdin.
+                    cd $(git rev-parse --show-toplevel)/packages/p2p
+                    echo "Uploading new strings to Crowdin"
+                    crowdin upload sources -T ${{ secrets.CROWDIN_API_KEY }}
+
+                    # Send a message to Slack (granted we have a webhook secret).
+                    # This check also allows a repo admin to disable the Slack message by removing the secret.
+                    if [ -n "${{ secrets.TRANSLATIONS_SLACK_WEBHOOK }}" ]; then
+                      echo "Sending message to Slack (#team_translations)"
+                      curl -X POST -H 'Content-type: application/json' --data '{"text":"There are new or updated strings available for DP2P (https://crowdin.com/project/deriv-app)."}' ${{ secrets.TRANSLATIONS_SLACK_WEBHOOK }}
+                    fi
+                  fi
+
+                  # Download latest translations from Crowdin
+                  cd $(git rev-parse --show-toplevel)/packages/translations
+                  echo "Attempting to download updated translations from Crowdin"
+                  crowdin download -T ${{ secrets.CROWDIN_API_KEY }}
+
+                  # Download latest translations from Crowdin in p2p
+                  cd $(git rev-parse --show-toplevel)/packages/p2p
+                  echo "Attempting to download updated translations from Crowdin"
+                  crowdin download -T ${{ secrets.CROWDIN_API_KEY }}
+
+                  if [ -z "$(git status --porcelain)" ]; then
+                    echo "Found no new translation files that need to be merged with master. Not creating a PR."
+                  else
+                    echo "Found updated translation files that need to be merged with master. Creating a PR."
+
+                    # Commit the newly downloaded files
+                    cd $(git rev-parse --show-toplevel)
+                    git add .
+                    git commit -m "translations: 📚 sync translations with crowdin"
+
+                    # Force push to this branch in case a previous run created it.
+                    git push --set-upstream origin "$branch_name" -f
+
+                    sudo apt install gh
+                    gh auth login --with-token <<< ${{ github.token }}
+                    gh pr close "$branch_name" || true
+                    gh pr create --fill --base "master" --head "binary-com:$branch_name"
+                  fi
```

Comment:
hi @ashraf-deriv, looks like your change is adding a lots of empty spaces.  only node version should be changed to 16. you can discard additional changes and better to use yaml linter or circleci cli tool for validating the config. `circleci config validate` should help from the root of repository

Reply:
It is because of the prettier code formatter, will revert the formatting.

Reply:
@ashraf-deriv it seems we still have the prettier changes on the file.

Reply:
@ali-hosseini-fs @shudarshon-binary Hi guys, The spaces are removed and all yaml validity checked using yaml linter.